### PR TITLE
aws scenario: .ssh -> .aws for AWS credentials, modifications for local running

### PR DESCRIPTION
**Pull request by zmerlynn**
Fix a typo in here, and setup the python script so you can run it
without bootstrap (otherwise setting --cluster and --store is a pain).

### CODE REVIEW

File: scenarios/kubernetes_aws.py [hunk 1]
```diff
@@ -95,18 +96,19 @@ def main(args):
...
       '-v', '%s:%s:ro' % (args.aws_pub, aws_pub),
       '-v', '%s:%s:ro' % (args.aws_cred, aws_cred),
-      '-v', '%s:%s:ro' % (args.service_account, service),
       '-e', 'JENKINS_AWS_SSH_PRIVATE_KEY_FILE=%s' % aws_ssh,
       '-e', 'JENKINS_AWS_SSH_PUBLIC_KEY_FILE=%s' % aws_pub,
-      '-e', 'JENKINS_AWS_CREDENTIALS_FILE=%s' % aws_cred,
-      '-e', 'GOOGLE_APPLICATION_CREDENTIALS=%s' % service])
+      '-e', 'JENKINS_AWS_CREDENTIALS_FILE=%s' % aws_cred])
+    if args.service_account:
+      cmd.extend(['-v', '%s:%s:ro' % (args.service_account, service),
```
Comment (krzyzacy) [inline scenarios/kubernetes_aws.py]: oh, 4 spaces here :-)

Comment (zmerlynn) [inline scenarios/kubernetes_aws.py]: Fixed, PTAL

**Comment by zmerlynn** [comment]
Okay, who set the verifier to verify the wrong indentation level!? @fejta? :)

File: scenarios/kubernetes_aws.py (hunk 1 above)
Comment (krzyzacy) [inline scenarios/kubernetes_aws.py]: No?

And the format here should probably be
```
cmd.extend([
  '-v', 'FOO',
  '-v', 'BAR'])
```

Comment (zmerlynn) [inline scenarios/kubernetes_aws.py]: I didn't add the dangling comma because the previous list was missing it, too.

**Comment by zmerlynn** [comment]
Oh, weird, the internal style guide and https://google.github.io/styleguide/pyguide.html#Indentation differ.

**Comment by krzyzacy** [comment]
:smiling_imp: So that we can classify internal code and external code!

File: scenarios/kubernetes_aws.py (hunk 1 above)
Comment (krzyzacy) [inline scenarios/kubernetes_aws.py]: I mean a new line after `([`? (Not 100% sure but I was told this way)

Eh seems lint did not shout so I assume it's fine..

**Pull request merged**